Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -125,7 +125,7 @@ extension RemoteDataServicesManager {
         dispatchQueue(for: remoteDataService, withRemoteDataType: .carb).async {
             let semaphore = DispatchSemaphore(value: 0)
             let queryAnchor = UserDefaults.appGroup?.getQueryAnchor(for: remoteDataService, withRemoteDataType: .carb) ?? CarbStore.QueryAnchor()
-
+            
```

Comment:
[nit] I try to avoid unnecessary whitespace changes since it just makes the relative delta larger. Don't worry about backing it out now, but consider catching them when doing your final review before committing or PR submit.